domains.txt

```diff
@@ -216,13 +249,15 @@ eurobase.com
 eurostaffgroup.com
 eurotimesolutions.co.uk
 eutopiaonline.com
+evermoreglobal.co.uk
 evodia.co.uk
 evolution-contract.co.uk
 evolution-jobs.co.uk
 evolution.training
 evolutionjobs.co.uk
 evolutionjobs.sg
 evosearch.co.uk
+exacityrecruitment.co.uk
```

Comment:
These are commonly heard complaints about Recruitment as an industry, I am sure many of you will have experienced these.
 
Not anymore.